sgn.conf

```diff
@@ -70,6 +70,9 @@ brapi_include_CO_xref 1
 
 allow_repeat_measures 0
 
+#export trait names as synonyms (1) or the original trait name (0)
```

Comment:
I think we need to also add this config item in the breedbase_configs deploy script

Reply:
@nickpalladino created: https://github.com/Breeding-Insight/breedbase_configs/pull/28